.github/workflows/release.yaml

```diff
@@ -44,15 +45,29 @@ jobs:
       - name: Install tool dependencies
         uses: jdx/mise-action@v2
 
-      - name: Package CRDs
+      - name: Package kubernetes manifests & CRDs
         run: |
+          # package just CRDs
           cat deploy/charts/prefect-operator/crds/*.yaml > prefect-crds.yaml
+          # handle helm dependencies
+          helm repo add bitnami https://charts.bitnami.com/bitnami
+          helm dependency build deploy/charts/prefect-operator
+          # template the helm chart including the CRDs
+          helm template prefect-operator deploy/charts/prefect-operator \
+          --include-crds --set operator.image.tag=${{ github.ref_name }} \
+          > prefect-operator.yaml
+          # Remove labels relevant only for Helm installs
+          yq -i 'del(.metadata.labels."app.kubernetes.io/managed-by")' prefect-operator.yaml
+          yq -i 'del(.metadata.labels."helm.sh/chart")' prefect-operator.yaml
+          yq -i 'del(.spec.template.metadata.labels."app.kubernetes.io/managed-by")' prefect-operator.yaml
+          yq -i 'del(.spec.template.metadata.labels."helm.sh/chart")' prefect-operator.yaml
+          # Ensure all references to app version match the released version tag
+          yq -i '(.. | select(tag == "!!str" and . == "v0.0.0")) |= "${{ github.ref_name }}"' prefect-operator.yaml
```

Comment:
This is looking good - tested with `dyff between` to confirm the differences (I should've put the `v` prefix but the concept applies).

```diff
@@ metadata.labels @@
# v1/ServiceAccount/default/prefect-operator
! - two map entries removed:
- app.kubernetes.io/managed-by: Helm
- helm.sh/chart: prefect-operator-v0.0.0

@@ metadata.labels.app.kubernetes.io/version @@
# v1/ServiceAccount/default/prefect-operator
! ± value change
- v0.0.0
+ 0.3.0

@@ metadata.labels @@
# rbac.authorization.k8s.io/v1/ClusterRole/prefect-operator
! - two map entries removed:
- app.kubernetes.io/managed-by: Helm
- helm.sh/chart: prefect-operator-v0.0.0

@@ metadata.labels.app.kubernetes.io/version @@
# rbac.authorization.k8s.io/v1/ClusterRole/prefect-operator
! ± value change
- v0.0.0
+ 0.3.0

@@ metadata.labels @@
# rbac.authorization.k8s.io/v1/ClusterRoleBinding/prefect-operator
! - two map entries removed:
- app.kubernetes.io/managed-by: Helm
- helm.sh/chart: prefect-operator-v0.0.0

@@ metadata.labels.app.kubernetes.io/version @@
# rbac.authorization.k8s.io/v1/ClusterRoleBinding/prefect-operator
! ± value change
- v0.0.0
+ 0.3.0

@@ metadata.labels @@
# rbac.authorization.k8s.io/v1/Role/default/prefect-operator:leaderelection
! - two map entries removed:
- app.kubernetes.io/managed-by: Helm
- helm.sh/chart: prefect-operator-v0.0.0

@@ metadata.labels.app.kubernetes.io/version @@
# rbac.authorization.k8s.io/v1/Role/default/prefect-operator:leaderelection
! ± value change
- v0.0.0
+ 0.3.0

@@ metadata.labels @@
# rbac.authorization.k8s.io/v1/RoleBinding/default/prefect-operator:leaderelection
! - two map entries removed:
- app.kubernetes.io/managed-by: Helm
- helm.sh/chart: prefect-operator-v0.0.0

@@ metadata.labels.app.kubernetes.io/version @@
# rbac.authorization.k8s.io/v1/RoleBinding/default/prefect-operator:leaderelection
! ± value change
- v0.0.0
+ 0.3.0

@@ metadata.labels @@
# apps/v1/Deployment/default/prefect-operator
! - two map entries removed:
- app.kubernetes.io/managed-by: Helm
- helm.sh/chart: prefect-operator-v0.0.0

@@ metadata.labels.app.kubernetes.io/version @@
# apps/v1/Deployment/default/prefect-operator
! ± value change
- v0.0.0
+ 0.3.0

@@ spec.template.metadata.labels @@
# apps/v1/Deployment/default/prefect-operator
! - two map entries removed:
- app.kubernetes.io/managed-by: Helm
- helm.sh/chart: prefect-operator-v0.0.0

@@ spec.template.metadata.labels.app.kubernetes.io/version @@
# apps/v1/Deployment/default/prefect-operator
! ± value change
- v0.0.0
+ 0.3.0
```